tiles/src/main/java/com/protomaps/basemap/layers/Boundaries.java

```diff
@@ -24,7 +24,6 @@ public String name() {
   public void processNe(SourceFeature sf, FeatureCollector features) {
     var sourceLayer = sf.getSourceLayer();
     var kind = "";
-    var kindDetail = "";
```

Comment:
`kindDetail` was implemented in v3 as a forward compatible change to Tilezen 1.9 schema. It's not necessary to remove it v4, although you can if you want. Natural Earth has a lot of extra fidelity that wasn't captured before... but for rendering probably just kind and the disputed boolean are sufficient for most basic maps.